tools/common/schema/setuptask.go

```diff
@@ -71,16 +75,27 @@ func (task *SetupTask) Run() error {
 		}
 	}
 
-	if len(config.SchemaFilePath) > 0 {
-		filePath, err := filepath.Abs(config.SchemaFilePath)
-		if err != nil {
-			return err
+	if len(config.SchemaFilePath) > 0 || len(config.SchemaName) > 0 {
+		var stmts []string
+		var err error
+		if len(config.SchemaName) > 0 {
+			fsys := dbschemas.Assets()
+			schemaFilePath := config.SchemaName + "/schema" + schemaFileEnding(config.SchemaName)
+			schemaBuf, err2 := fs.ReadFile(fsys, schemaFilePath)
+			if err2 != nil {
+				return fmt.Errorf("error reading file %s: %w", schemaFilePath, err)
+			}
+			stmts, err = persistence.LoadAndSplitQueryFromReaders([]io.Reader{bytes.NewBuffer(schemaBuf)})
+		} else {
+			filePath, err2 := filepath.Abs(config.SchemaFilePath)
+			if err2 != nil {
+				return err
+			}
+			stmts, err = persistence.LoadAndSplitQuery([]string{filePath})
```

Comment:
I didn't change this to the `os.DirFS(dirPath)` method, because I didn't want to assume that SchemaFilePath had any directory levels, whereas for `SchemaDir` we know it's a directory